flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/connector/datagen/table/RandomGeneratorVisitor.java

```diff
@@ -203,6 +217,33 @@ public DataGeneratorContainer visit(DayTimeIntervalType dayTimeIntervalType) {
                 RandomGenerator.longGenerator(config.get(min), config.get(max)), min, max);
     }
 
+    @Override
+    public DataGeneratorContainer visit(TimestampType timestampType) {
```

Comment:
nit: should we move the override implementation to base class `DataGenVisitorBase`, or remove the default implementation in `DataGenVisitorBase`? IMO, datagen for timestamp type only works for random.

Reply:
I've remove the default implementation in `DataGenVisitorBase`.